### PR TITLE
docs: Updated the instructions for installing Atom and Flutter package

### PR DESCRIPTION
**Pull request by qchong**
@HansMuller

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -119,22 +119,28 @@ you want Flutter to use a different installation of the Android SDK, you must se
...
 
-Download Atom from [atom.io](https://atom.io/) and then install the
-[Flutter package](https://atom.io/packages/flutter). If you need help installing
-packages, consult the [Atom documentation](https://atom.io/docs/v1.3.2/using-atom-atom-packages)
-or [email us](flutter-dev@googlegroups.com).
+To install Atom:
+
+1. Download Atom from the [atom.io](https://atom.io/) site.
+2. Extract the Atom executable from the downloaded file.
+
```
Comment (HansMuller) [inline setup.md]: I think you still need to say "install the flutter package".

Comment (HansMuller) [inline setup.md]: Oops, I see, the package install step is under "Configure Atom...". So maybe the "if you need help installing..." item should go below?

**Comment by HansMuller** [comment]
LGTM

File: setup.md (hunk 1 above)
Comment (qchong) [inline setup.md]: Thanks, I've made the change you suggested and re-pushed.

**Pull request merged**